lib/trailblazer/macro/each.rb

```diff
@@ -0,0 +1,44 @@
+require "securerandom"
+
+module Trailblazer
+  module Macro
+    def self.Each(source:, target:, id: "Each/#{SecureRandom.hex(4)}", &block)
+      activity = Class.new(Trailblazer::Activity::Railway, &block)
+      wrapped = Wrapper.new(activity, source, target)
```

Comment:
@btolarz Can we use `Wrap` macro instead of defining the `Wrapper` ? It only handles `Left` and `Right` signals but `Wrap` handles all registered signals.
 
Regarding the presence of a `target` in `ctx`, it can be handled via `input/output` options.